packages/protractor/protractor_web_test.bzl

```diff
@@ -110,6 +110,15 @@ def _protractor_web_test_impl(ctx):
     )
 
     runfiles = [configuration] + configuration_sources + on_prepare_sources
+    server_files = depset()
+
+    # If a server has been specified, add it to the runfiles together with it's required runfiles. This is necessary
+    # as the test executable references the server executable as per `TMPL_server` and executes it.
+    if ctx.executable.server:
+        server_files = depset(
```

Comment:
nit : I would have named it `server_runfiles` but not important enough to fix

Reply:
yup. very true. I will send a follow-up!